engine/clients/pgvector/parser.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, List, Optional
+
+from engine.base_client.parser import BaseConditionParser, FieldValue
+
+
+class PgVectorConditionParser(BaseConditionParser):
+    def build_condition(
+        self, and_subfilters: Optional[List[Any]], or_subfilters: Optional[List[Any]]
+    ) -> Optional[Any]:
+        raise NotImplementedError()
+
+    def build_exact_match_filter(self, field_name: str, value: FieldValue) -> Any:
+        raise NotImplementedError()
+
+    def build_range_filter(
+        self,
+        field_name: str,
+        lt: Optional[FieldValue],
+        gt: Optional[FieldValue],
+        lte: Optional[FieldValue],
+        gte: Optional[FieldValue],
+    ) -> Any:
+        raise NotImplementedError()
+
+    def build_geo_filter(
+        self, field_name: str, lat: float, lon: float, radius: float
+    ) -> Any:
+        raise NotImplementedError()
```

Comment:
Why are all of these marked as not yet implemented at this time?

Reply:
We just want to update [Single Node Benchmarks](https://qdrant.tech/benchmarks/#single-node-speed-benchmark) for now. The datasets involved don't have any filters. It's all embeddings. Hence these aren't required.

Reply:
Just implemented all the functions in `PgVectorConditionParser`